test/test_quantized.py

```diff
@@ -210,6 +210,27 @@ def test_max_pool2d(self, Q, kernel, stride, dilation, padding):
     " we are in a UBSAN environment.",
 )
 class TestQuantizedLinear(unittest.TestCase):
+    def test_dynamic_qlinear(self):
+        batch_size = 1
+        input_channels = 2
+        output_channels = 2
+        X = torch.rand(batch_size, input_channels).float()
+        W = torch.rand(output_channels, input_channels).float()
+        b = torch.rand(output_channels).float()
+
+        # Weight prepacking operator for quantized Linear
+        W_int8, col_offsets, scale, zero_point = torch.fbgemm_linear_quantize_weight(W)
+        W_prepack = torch.fbgemm_pack_quantized_matrix(W_int8.clone(), W_int8.size(1), W_int8.size(0))
+        # Quantized Linear operator with prepacked weight
+        Y_dq = torch.fbgemm_linear_int8_weight(
```

Comment:
NIT: Can we have better names for the fbgemm operators? fbgemm_linear_int8_weight-> fbgemm_linear_int8_weight_fp32_activation? This name does not imply whether we are doing static or dynamic quant.

Reply:
Fixed in https://github.com/pytorch/pytorch/pull/22955.